spaceflights-pandas/hooks/post_gen_project.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+from kedro.templates.project.hooks.utils import (
+
+    setup_template_add_ons,
+    sort_requirements,
+)
+from kedro.framework.cli.starters import _parse_add_ons_input
```

Comment:
```suggestion
from kedro.framework.cli.starters import _parse_add_ons_input
```

Having a private function imported outside of `kedro` is a red flag to me. We should remove this for other starters as well, as it doesn't seem to get call at all?

Reply:
Thanks, yes, looks like some legacy, I copied the whole file and missed it. I will fix that for all 4 starters.